### PR TITLE
setup: pin viv-utils version

### LINKED ISSUE #419: setup: bump viv-utils to v0.3.18
https://github.com/williballenthin/viv-utils/releases/tag/v0.3.18

### PR DESCRIPTION
**Pull request by williballenthin**
closes #419

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -33,7 +33,7 @@
     requirements.append("enum34==1.1.6")  # v1.1.6 is needed by halo 0.0.30 / spinners 0.0.24
     requirements.append("halo==0.0.30")  # halo==0.0.30 is the last version to support py2.7
     requirements.append("vivisect==0.1.0")
-    requirements.append("viv-utils")
+    requirements.append("viv-utils==0.3.18")
```
Comment (Ana06) [inline setup.py:36]: Should we use `>=` instead of `==`? 🤔

**Review by Ana06 — APPROVED** [review]

File: setup.py (hunk 1 above)
Comment (williballenthin) [reply setup.py:36]: i think its better to be explicit here, esp with dependabot configured

Comment (mr-tz) [reply setup.py:36]: agreed, only need a PyPi release

**Comment by Ana06** [comment]
Tests are failing because the upload of viv-utils 0.3.18 to pypi failed. This PR should fix it: https://github.com/williballenthin/viv-utils/pull/14

**Comment by williballenthin** [comment]
viv-utils v0.3.19 released and updated here @Ana06

**Pull request merged**